src/main/java/com/mineinabyss/bonfire/listeners/PlayerListener.kt

```diff
@@ -192,6 +195,13 @@ object PlayerListener : Listener {
                 }
             }
         }
+
+        transaction {
```

Comment:
Can include this code in the transaction above.

Reply:
What about the ?: return@transaction

Reply:
oh ye, hm. could just change that to an if statement around the code below, with a null check. I'd prefer that than starting two separate transactions

Reply:
This is not possible.
At least from what Scyu and I tested.